"Undefined symbols for architecture arm64"

My environment:
MacOS Big Sur 11.2.1 (20D74)
Xcode Version 12.4 (12D4e)

full error:
```
Undefined symbols for architecture arm64:
  "___llvm_profile_runtime", referenced from:
      ___llvm_profile_runtime_user in libTestFairy.a(TFFeedbackDrawView.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFDelegateImpl.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFMotionEndedClassVisitor.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFAsn1Parser.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFRSAUtils.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFPerformanceUtils.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFAESUtils.o)
      ...
     (maybe you meant: ___llvm_profile_runtime_user)
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

i have tracked this down to be caused by the [latest 1.28.6 Spec published](https://github.com/CocoaPods/Specs/commits/master/Specs/b/0/e/TestFairy)

since the [plugin.xml](https://github.com/testfairy/testfairy-cordova-plugin/blob/master/plugin.xml#L25) has `spec="~> 1.28.2"`, it started using the broken spec. if you remove the `~>` to explicitly set the version, it works fine. i tested with spec version 1.28.5 and it was working ok for me, seems that 1.28.6 is the problematic one.

im not sure if we should explicitly set the spec version in this plugin.xml, or if there is an actual issue with the podspec, or maybe both. im not sure how to report an issue abotu the podspec, so im reporting it here


fyi, you may also want to consider moving away from using the `<framework>` tag and instead use `<podspec>`, as im getting this warning from cordova about it: **"framework" tag with type "podspec" is deprecated and will be removed. Please use the "podspec" tag**. just note that this would also force your users to use cordova-ios@5 and cordova@9

example working diff:
```
diff --git a/testfairy-cordova-plugin/plugin.xml b/testfairy-cordova-plugin/plugin.xml
index 60856846dc0..55b2ac70083 100755
--- a/testfairy-cordova-plugin/plugin.xml
+++ b/testfairy-cordova-plugin/plugin.xml
@@ -22,7 +22,14 @@
         </config-file>
         <header-file src="src/ios/CDVTestFairy.h" />
         <source-file src="src/ios/CDVTestFairy.m" />
-        <framework src="TestFairy" type="podspec" spec="~> 1.28.2" />
+        <podspec>
+            <config>
+                <source url="https://cdn.cocoapods.org/"/>
+            </config>
+            <pods use-frameworks="true">
+                <pod name="TestFairy" spec="1.28.5" />
+            </pods>
+        </podspec>
     </platform>
     <platform name="android">
         <source-file src="src/android/CDVTestFairy.java" target-dir="src/com/testfairy" />
```

@vijaysharm  I can reproduce the same error, too.

Reproduce steps:
```
pod repo update
npx cordova create SampleApp com.sample.app SampleApp
cd SampleApp/
npx cordova platform add ios
npx cordova plugin add com.testfairy.cordova-plugin
npx cordova build ios --simulator
```

It will throw
```
Undefined symbols for architecture x86_64:
  "___llvm_profile_runtime", referenced from:
      ___llvm_profile_runtime_user in libTestFairy.a(TFFeedbackDrawView.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFDelegateImpl.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFMotionEndedClassVisitor.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFAsn1Parser.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFRSAUtils.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFPerformanceUtils.o)
      ___llvm_profile_runtime_user in libTestFairy.a(TFAESUtils.o)
      ...
     (maybe you meant: ___llvm_profile_runtime_user)
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I compared `libTestFairy.a` between 1.28.5 and 1.28.6.

`1.28.5` does not return undefined symbol `___llvm_profile_runtime`.

```
TestFairySDK-1.28.5$ nm -Au libTestFairy.a | grep ___llvm_profile_runtime
TestFairySDK-1.28.5$
```

However, `1.28.6` returns undefiend sysmbol  `___llvm_profile_runtime`.
```
TestFairySDK-1.28.6$ nm -Au libTestFairy.a | grep ___llvm_profile_runtime
libTestFairy.a:TFFeedbackDrawView.o: ___llvm_profile_runtime
libTestFairy.a:TFNewRelicFeature.o: ___llvm_profile_runtime
libTestFairy.a:TFDelegateImpl.o: ___llvm_profile_runtime
libTestFairy.a:TFInstalledFrameworkClassVisitor.o: ___llvm_profile_runtime
libTestFairy.a:TFMotionEndedClassVisitor.o: ___llvm_profile_runtime
libTestFairy.a:TFAsn1Parser.o: ___llvm_profile_runtime
...
TestFairySDK-1.28.6$
```

So, I think the 1.28.6 library was built with code coverage enabled.

https://developer.apple.com/library/archive/documentation/DeveloperTools/Conceptual/testing_with_xcode/chapters/07-code_coverage.html

If I turn on the code coverage in the project, the error does not happen. However, the cordova build does not turn on code coverage by default, `libTestFairy.a` should be fixed.

**code coverage disabled:**
<img width="1193" alt="code coverage disabled" src="https://user-images.githubusercontent.com/13875691/116322784-9baecf80-a771-11eb-875b-750370d632d9.png">


**code coverage enabled:**
<img width="1157" alt="code coverage enabled" src="https://user-images.githubusercontent.com/13875691/116322817-af5a3600-a771-11eb-9959-a6cd64de0799.png">

--- a/.travis.yml
+++ b/.travis.yml
@@ -13,6 +13,7 @@ before_install:
 - echo yes | android update sdk --filter platform-tools --no-ui --force > /dev/null
 - echo yes | android update sdk --filter android-28 --no-ui --force > /dev/null
 - echo yes | android update sdk --filter build-tools-28.0.3 --all --no-ui --force > /dev/null
+- pod repo update
 install:
 - npm install -g cordova
 - npm install -g ionic

--- a/package.json
+++ b/package.json
@@ -2,6 +2,9 @@
   "name": "com.testfairy.cordova-plugin",
   "version": "2.48.0",
   "description": "TestFairy SDK plugin for Cordova",
+  "engines": {
+    "node": ">=6.0.0"
+  },
   "cordova": {
     "id": "com.testfairy.cordova-plugin",
     "platforms": [

--- a/plugin.xml
+++ b/plugin.xml
@@ -4,7 +4,7 @@
     <description>TestFairy SDK plugin for Cordova</description>
     <license>Apache 2.0 License</license>
     <engines>
-        <engine name="cordova" version=">=3.2.0" />
+        <engine name="cordova" version=">=9.0.0" />
     </engines>
 
     <js-module src="www/testfairy.js" name="TestFairy">
@@ -22,7 +22,14 @@
         </config-file>
         <header-file src="src/ios/CDVTestFairy.h" />
         <source-file src="src/ios/CDVTestFairy.m" />
-        <framework src="TestFairy" type="podspec" spec="1.28.7" />
+        <podspec>
+            <config>
+                <source url="https://cdn.cocoapods.org/"/>
+            </config>
+            <pods use-frameworks="true">
+                <pod name="TestFairy" spec="1.28.7" />
+            </pods>
+        </podspec>
     </platform>
     <platform name="android">
         <source-file src="src/android/CDVTestFairy.java" target-dir="src/com/testfairy" />

